badge_server/badge_server.py

```diff
@@ -222,7 +284,44 @@ def index():
     return 'Visitor number: {}'.format(value)
 
 
-@app.route('/self_compatibility_badge/image')
+@app.route('/one_badge_image')
+def one_badge_image():
+    package_name = flask.request.args.get('package')
+    url_prefix = flask.request.url_root[:-1]
+    # Call the url for each badge to run the checks.
```

Comment:
... the checks. This will populate the individual caches, which are used to calculate the final image state.

Reply:
Done.